### ISSUE TITLE
Navigationbar invisible until window is resized on sample

### ISSUE THREAD

**Issue by Levi--G** [unoplatform/uno.toolkit.ui#1180]
## Current behavior

I am running a fresh sample with following options:
-.NET 8
-Only webassembly and SKIA
-MVVM
-XAML
-Fluent + theme service
-no config, http or localisation
-+Server + wasm multithreading no PWA and no VSCODE
-default rest of options

The navigationbar after going to the second page does not show on desktop/windows (skia) until the window is resized or switched to another display. InvalidateMeasure seems to have no impact.

I noticed the issue is also affected by binding the navigationbar content. If it is binded it works (like main page) but if its a constant text it will not show the entire bar (including primary and secondary icons) although the buttons do still work.

Works:
`<utu:NavigationBar Content="{Binding Title}"/>`
Does not work
`<utu:NavigationBar Content="Second Page"/>` 

## Expected behavior

Navigationbar should be shown right away

## How to reproduce it (as minimally and precisely as possible)

Either use the zip or create a project with above settings, the sample will not show a navbar on the second page. Only desktop windows was tested.

[UnoApp1.zip](https://github.com/user-attachments/files/16112283/UnoApp1.zip)

Nuget Package: Uno.WinUI

Package Version(s): 5.2.175

Affected platform(s):

- [ ] WebAssembly
- [ ] Android
- [ ] iOS
- [ ] macOS (AppKit)
- [ ] Mac Catalyst
- [ ] Skia
  - [x] WPF
  - [ ] GTK (Linux)
  - [ ] Linux Framebuffer
  - [ ] Tizen
- [ ] Windows

IDE:

- [x] Visual Studio 2022
- [ ] Visual Studio 2019
- [ ] Visual Studio Code
- [ ] Visual Studio for Mac
- [ ] Rider Windows
- [ ] Rider macOS

Relevant plugins:

**Comment by kazo0** [comment]
It seems that this is fixed as of package version 5.3.0-dev.12 of Uno. With the upcoming 5.3 stable release, it should include the fix in the next stable packages

**Comment by VincentH-Net** [comment]
I ran into this issue on WASM with the latest Uno stable release using OOB Fluent template choice from Uno project template. Actually it seems like multiple issues:

- When the browser window is made wider while on another page than the home page, then on returning to the home page the header container is less wide than the page, causing primarycommands be clipped and so disappear, and the background color to be clipped as well.
- The show more "..." is always visible, even when there is no overflow. 
- Not clear how to use fluent styles are not used, no theming support (dark / light ...)
- Content is not horizontally centered when using PrimaryCommands or when no back button visible, requires brittle workarounds with margins

I Hope that this control can receive some love as it is both complex to build well for all platforms and needed in almost every app

**Comment by kazo0** [comment]
Yes the Toolkit controls are currently most useful with the Material theme, we definitely need to focus on the default Fluent style for the controls.

Starting an issue to track this: https://github.com/unoplatform/uno.toolkit.ui/issues/1325
